lib/app-wrapper.js

```diff
@@ -2,12 +2,25 @@ const path = require('path');
 const ElmCommunication = require('./elm-communication');
 const loadCompiledElmApp = require('./load-compiled-app');
 
+/**
+ * @typedef { import('./types/options').Options } Options
+ * @typedef { import('./types/options').ReviewOptions } ReviewOptions
+ * @typedef { import('./types/app').App } App
+ */
```

Comment:
Once ts 5.5 is stable (which should be soon), we can switch to the import syntax instead here:
```suggestion
/**
 * @import {Options} from './types/options';
 * @import {ReviewOptions} from './types/options';
 * @import {App} from './types/app';
 */
```

Reply:
> soon

Today!
I'm currently cleaning up this branch locally, I should have something up today or tomorrow (certainly tomorrow in Europe. How's the future? 😉).